book_source/03_topical_pages/06_data/01_meteorology.Rmd

```diff
@@ -132,3 +132,11 @@ Resolution: 30 min
 Availability: Varies by [site](https://meta.icos-cp.eu/collections/q4V7P1VLZevIrnlsW6SJO1Rz)
 
 Notes: To use this option, set `source` as `ICOS` and a `product` tag containing `etc` in `pecan.xml`
+
+## ECMWF Open Data
+
+Scale: Global
+
+Resolution: 6 hour, 0.4 degree
```

Comment:
document that current implementation only uses forecast hour 00 when download function is called from within the PEcAn workflow